src/neo-vm/OpCode.cs

```diff
@@ -2,142 +2,460 @@ namespace Neo.VM
 {
     public enum OpCode : byte
     {
+      /// <summary>
+      ///
```

Comment:
👎  xD

Reply:
hahah that was my reference for copy and paste... forgot it on the final version :D